### PR TITLE
MUI Dividers conversion 

### PR DESCRIPTION
**Pull request by d88naimi**

### CODE REVIEW

File: src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ContentInsights/ContentInsights.js [hunk 1]
```diff
@@ -299,7 +300,12 @@ export function ContentInsights(props) {
             Unique Words* <span>{totalUniqueNonCommonWords}</span>
           </div>
         </div>
-        <Divider className={styles.divider} />
+        <Divider
+          sx={{
+            mt: 1,
```
Comment (agalin920) [inline src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ContentInsights/ContentInsights.js:305]: Can you simplify this to my: 1 please

File: src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ContentInsights/ContentInsights.js [hunk 2]
```diff
@@ -318,9 +324,14 @@ export function ContentInsights(props) {
             }
           })}
         </div>
-        <Divider className={styles.divider} />
+        <Divider
+          sx={{
+            mt: 1,
```
Comment (agalin920) [inline src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ContentInsights/ContentInsights.js:329]: Can you simplify this to my: 1 please

File: src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ContentInsights/ContentInsights.js [hunk 3]
```diff
@@ -368,8 +379,13 @@ export function ContentInsights(props) {
             })}
           </div>
         )}
-        <Divider className={styles.divider} />
-        <h4>Words Occurences from the URL, Meta Title and Description</h4>
+        <Divider
+          sx={{
+            mt: 1,
```
Comment (agalin920) [inline src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ContentInsights/ContentInsights.js:384]: Can you simplify this to my: 1 please

File: src/apps/settings/src/app/views/Instance/Instance.js [hunk 4]
```diff
@@ -144,11 +145,27 @@ export default connect((state) => {
           </div>
         </div>
       </div>
-      <Divider />
+      <Divider
+        sx={{
+          mt: 1,
+          mr: 2,
```
Comment (agalin920) [inline src/apps/settings/src/app/views/Instance/Instance.js:151]: Can you simplify this to my: 1 mx: 2 please

File: src/apps/settings/src/app/views/Instance/Instance.js [hunk 5]
```diff
@@ -144,11 +145,27 @@ export default connect((state) => {
...
+        }}
+      />
       <div className={styles.row}>
         {fields.map((field) => {
           {
-            i % 2 == 0 && <Divider />;
+            i % 2 == 0 && (
+              <Divider
+                sx={{
+                  mt: 1,
```
Comment (agalin920) [inline src/apps/settings/src/app/views/Instance/Instance.js:162]: Can you simplify this to my: 1 mx: 2 please

**Review by agalin920 — CHANGES_REQUESTED** [review]
See comments for changes

**Review by agalin920 — APPROVED** [review]
Approved

File: src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ContentInsights/ContentInsights.js [hunk 6]
```diff
@@ -368,8 +377,12 @@ export function ContentInsights(props) {
...
           </div>
         )}
-        <Divider className={styles.divider} />
-        <h4>Words Occurences from the URL, Meta Title and Description</h4>
+        <Divider
+          sx={{
+            my: 1,
+          }}
+        />
+        <h4>Words Occurrences from the URL, Meta Title and Description</h4>
```
[the remark this answers is not in the document]
Comment (shrunyan) [reply src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ContentInsights/ContentInsights.js:385]: also lower case "occurrences"

**Review by shrunyan — CHANGES_REQUESTED** [review]

**Review by shrunyan — APPROVED** [review]

**Comment by shrunyan** [comment]
@d88naimi there is a test failing for `releases/members.spec.js`. Would you investigate what's happening there

File: cypress/integration/content/headTags_spec.js [hunk 7]
```diff
@@ -13,9 +13,9 @@ describe("Head Tags", () => {
 
     cy.get("[data-cy=contentNavButton]").click();
 
-    cy.get("[data-cy=head]").click();
+    cy.get("[data-cy=head]").click({ force: true });
```
Comment (d88naimi) [inline cypress/integration/content/headTags_spec.js:16]: contentNav wont close  event with `cy.get("[data-cy=contentNavButton]").click();` and is blocking the click of Head Tag button that's why I adde force true

**Comment by d88naimi** [comment]
Made a new PR here https://github.com/zesty-io/manager-ui/pull/1111

**Pull request closed without merging**